packages/eas-cli/src/project/ios/target.ts

```diff
@@ -244,6 +256,32 @@ export function getApplePlatformFromSdkRoot(target: Target): ApplePlatform {
   } else if (sdkRoot.includes('macosx')) {
     return ApplePlatform.MAC_OS;
   } else {
+    return null;
+  }
+}
+
+export function getApplePlatformFromDeviceFamily(target: Target): ApplePlatform | null {
+  const deviceFamily = target.buildSettings?.TARGETED_DEVICE_FAMILY;
+  if (!deviceFamily) {
+    return null;
+  }
+  if (typeof deviceFamily === 'string') {
+    const deviceFamilyNumber = parseInt(deviceFamily, 10);
```

Comment:
After applying Doug's comment (https://github.com/expo/eas-cli/pull/1417/files#r984991660) I think it's better to change it so that we don't interpret "1,2" as "1".
<img width="187" alt="Screenshot 2022-10-03 at 11 27 21" src="https://user-images.githubusercontent.com/5256730/193544553-07bf949d-25ad-4415-aa2f-f33ae2247a16.png">

More general comment: it's good practice to avoid using `parseInt` as the algorithm is not as obvious as people think. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt. Use `Number("some string")` if you need to convert a string to a number.